docs/src/main/sphinx/connector/hive.rst

```diff
@@ -398,6 +398,17 @@ Property Name                                      Description
                                                    Set to ``false`` to disable statistics. Disabling statistics
                                                    means that :doc:`/optimizer/cost-based-optimizations` can
                                                    not make smart decisions about the query plan.
+
+``hive.auto-purge``                                Sets default value for managed table property auto_purge`.    ``false``
+                                                   You can use this property to set auto_purge at catalog level
+                                                   and skip setting auto_purge table property during managed
+                                                   table creation.
+                                                   auto_purge managed table property when set to true overrides
```

Comment:
This info belongs to the table properties table .. expand there if necessary .. dont repeat here